e2e-tests/init-deploy/00-assert.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+timeout: 120
```

Comment:
is it necessary to wait up to 2 mins?

Reply:
kuttl tries to assert and if there are assertion failures it sleeps a second and assert again until the timeout. we're saying fail the test if you can't assert this in 2 mins, so it's a worst case scenario.

I tuned the timeouts in https://github.com/percona/percona-server-mysql-operator/pull/21/commits/f7e76c345bd7838fa30efc2b5eb5215b0eeb96ee